db/seeds.rb

```diff
@@ -5,3 +5,37 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+User.create!(email: 'Test@Fake.com', password: 'password123', sign_in_count: 0)
```

Comment:
your seeders should always be rerunnable. If I run this a second time, it will raise an exception because the record already exists. Utilize `find_by!` to make this script rerunnable

Also, you should only seed fake data in development so you should have some kind of flow control like this:

```erb
<%- if Rails.env.development? -%>
```